GithubPractice/ViewController.swift

```diff
@@ -20,8 +20,10 @@ class ViewController: UIViewController {
     
     private let defaults = UserDefaults.standard
     
-    let jobKey = "Job"
-    let birthDayKey = "BirthDay"
+    let firstNameKey = Constants.firstNameKey
+    let lastNameKey = Constants.lastNameKey
+    let jobKey = Constants.jobKey
+    let birthDayKey = Constants.birthDayKey
```

Comment:
it is a way of doing but I see no point of redeclaring these variables. you could use Constants.key right where you need it